src/main/java/com/gmail/nossr50/listeners/PlayerListener.java

```diff
@@ -423,11 +423,12 @@ public void onPlayerRespawn(PlayerRespawnEvent event) {
     public void onPlayerInteractLowest(PlayerInteractEvent event) {
         Player player = event.getPlayer();
 
-        if (event.getHand() != EquipmentSlot.HAND || !UserManager.hasPlayerDataKey(player) || player.getGameMode() == GameMode.CREATIVE) {
+        McMMOPlayer mcMMOPlayer = UserManager.getPlayer(player);
```

Comment:
I won't be able to accept the commit with this line here. If you see the if statement you moved to below it, one of the conditions is that mcMMO has data for the player. There have been issues in the past where the plugin failed to load an existing player and overwrote them with a new player with empty skills. (Usually caused by using an SQL server hosted elsewhere.)

It would be advisable to just leave the initial check as is and then add a new check for the mcability flag afterward.